index.html

```diff
@@ -86,8 +87,8 @@ <h2 class="head2">Tonic</h2>
                         <div class="link3">javaScript</div>
                     </li>
                 </ul>
-                <button class="see-project1 unique2">
-                    <div class="see-project">See Project</div>
+                <button class="see-project1 project-buttons unique2">
```

Comment:
Hello team! Good job, but please make sure all the buttons are working properly as the previous reviewer requested, and make sure you created all cards created dynamically.
![Screen Recording 2022-12-24 at 1 00 03 AM](https://user-images.githubusercontent.com/25887253/209409585-1c214c59-8f5f-42ee-8c47-40fb80de125f.gif)

[OPTIONAL]  According to the project requirement [here](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md#:~:text=You%20need%20to%20use,when%20the%20page%20loads.), the lines between 60-228 need to be removed and should be generated dynamically with JavaScript.

Reply:
@huseyinbiyik Hello,

Lines 60 - 228 are for the normal window web page and not for the pop-up window. Am I expected to remove them too?

Reply:
Hello,
No only the cards in the middle. You need to have a container wrapping the cards but all the rest need to be created dynamically with JavaScript. You can check my portfolio if you need an example 👍